index.html

```diff
@@ -8,6 +8,37 @@
     <title>Personal Portfolio</title>
 </head>
 <body>
-    
+    <nav class="toolbar">
+        <figure class="logo-wrapper">
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
+
+        <figure class="menu-icon">
+            <img src="./images/Union.png" alt="">
+        </figure>
```

Comment:
**REQUIRED:** The hamburger button should have `nav` tag, because you will use it for links to different parts of the page in the future.